src/System.Net.WebSockets/src/System/Net/WebSockets/WebSocketException.cs

```diff
@@ -122,11 +122,6 @@ public WebSocketException(string message, Exception innerException)
         {
         }
 
-        public override void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
@davidsh this doesn't match the description for this PR. (This is effectively _removing_ the test which runs fine in CoreFX and .Net Framework as far as we know.)